guide/ops/troubleshooting/increase-system-resource-limits.md

```diff
@@ -4,6 +4,16 @@ title: Increase System Resource Limits
 toc: /guide/toc.json
 ---
 
+If you encounter the following error:
+
+    Caused by: java.io.IOException: Too many open files
+            at java.io.UnixFileSystem.createFileExclusively(Native Method)[:1.8.0
+
+Please check limit for opened files `cat /proc/sys/fs/file-max` and increase it.
+You can increase limit for maximum opened files by setting in `/etc/sysctl.conf` `fs.file-max = 700000`
```

Comment:
You can increase the maximum limit of opened files by setting `fs.file-max` in `/etc/sysctl.conf`